src/AxelarProxy.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import { AxelarExecutable } from "lib/axelar-gmp-sdk-solidity/contracts/executable/AxelarExecutable.sol";
+import { Address } from "@openzeppelin/contracts/utils/Address.sol";
+
```

Comment:
Can you add a description to the contract outlining how it will work with the Sommelier protocol? In my bigger message at the end of this review, I outline how I perceive your code in this PR is to be carried out, but descriptions would help. It wouldn't hurt for the Auditor too :)